RT/dt_evolve_hpsi.f90

```diff
@@ -61,7 +62,7 @@ subroutine dt_evolve_hpsi
     call update(zfac,ztpsi(:,:,tid),zu(:,ib,ik))
 
 #ifdef ARTED_CURRENT_OPTIMIZED
-    call current_omp_KB_ST(ib,ik,zu(:,ib,ik))
+    if(flag_current) call current_omp_KB_ST(ib,ik,zu(:,ib,ik))
```

Comment:
Thank you for reviewing this pull request.
Indeed, this propagator reduces the computational cost (please see the slide of developer mailing list on 2016/12/22).

I also compared the computational cost with and without this `if` statement. And I confirmed the `if` statement reduces the computational cost.
I agree with you that  `if` statement may harm performance. Probably, we need two subroutines of "dt_evolve_hpsi"; One includes a part of current computation, and the other does not.